tasty-plutus/src/Test/Tasty/Plutus/Internal/Context.hs

```diff
@@ -198,15 +198,20 @@ compileSpending conf cb d val =
 compileMinting ::
   TransactionConfig ->
   ContextBuilder 'ForMinting ->
+  Value ->
   ScriptContext
-compileMinting conf cb =
+compileMinting conf cb val =
   ScriptContext go
     . Minting
     . testCurrencySymbol
     $ conf
   where
     go :: TxInfo
-    go = baseTxInfo conf cb
```

Comment:
Does this actually affect minting policy behaviour? I assumed this was only needed for validator scripts.